### PR TITLE
Added new function to check for isPredOrFunc

### PR DESCRIPTION
**Pull request by dalefrancis88**

### CODE REVIEW

File: src/core/isPredOrFunc.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const test = require('tape')
+
+const Pred = require('../Pred')
+const isNumber = require('./isNumber')
+const unit = require('./_unit')
+const isLargeNumber =
```
Comment (evilsoft) [inline src/core/isPredOrFunc.spec.js]: insert a new line above "block"ish bits of text please. Also in test 🌽structs should be places after all requires.

File: src/core/isPredOrFunc.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/** @license ISC License (c) copyright 2018 original and current authors */
+/** @author Dale Francis (dalefrancis88) */
+
+const Pred = require('./types').proxy('Pred')
+const isFunction = require('./isFunction')
+const isSameType = require('./isSameType')
+
+// isPredOrFunc :: (func | pred) -> bool
```
Comment (evilsoft) [inline src/core/isPredOrFunc.js]: should read like:
```js
// isPredOrFunc :: ((a -> b) | Pred a) -> Boolean
```

File: src/logic/and.js [hunk 3]
```diff
@@ -1,16 +1,13 @@
-/** @license ISC License (c) copyright 2017 original and current authors */
+/** @license ISC License (c) copyright 2018 original and current authors */
```
Comment (evilsoft) [inline src/logic/and.js]: This was added in 2017, should keep the original year

Comment (dalefrancis88) [reply src/logic/and.js]: This is reverted now

**Comment by evilsoft** [comment]
image

**Pull request merged**